Rakefile

```diff
@@ -0,0 +1,150 @@
+# Copyright 2015, SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "git"
+require "octokit"
+require "netrc"
+require "yaml"
+
+
+task :configure do
+  config = YAML.load_file(File.join(File.dirname(__FILE__), "config.yml"))
+  @all_barclamps = config["barclamps"]["crowbar"] + config["barclamps"]["openstack"]
```

Comment:
That sucks, there is no option for adding more barclamps to the config. You should write it different with ```collect``` or something like that.